packages/ui/src/components/Table/STableCard.vue

```diff
@@ -0,0 +1,187 @@
+<script setup lang="ts">
+import { TableActionColumnApi, TableColumnApi, TableRow } from '@/components'
+import { IconArrowsChevronDownRounded24, IconArrowRight16 } from '@/components/icons'
+import {
+  isDefaultColumn,
+  isExpandColumn,
+  isDetailsColumn,
+  getDefaultCellValue,
+  getCellTooltipContent,
+  isSelectionColumn,
+} from '@/components/Table/utils'
+import SButton from '@/components/Button/SButton.vue'
+
+const props = withDefaults(
+  defineProps<{
+    row: { data: TableRow; index: number }
+    columns?: (TableColumnApi | TableActionColumnApi)[]
+    activeExpandColumn?: (TableActionColumnApi & { type: 'expand' }) | null
+    expanded?: boolean
+    selected?: boolean
+    selectable?: boolean
+  }>(),
+  {
+    columns: () => [],
+    activeExpandColumn: null,
+    expanded: false,
+    selected: false,
+    selectable: false,
+  },
+)
+
+/* eslint-disable @typescript-eslint/unified-signatures */
+const emit = defineEmits<{
+  (event: 'select'): void
+  (event: 'expand'): void
+  (event: 'click:details'): void
+  (event: 'mouse-event:label', value: { column: TableColumnApi | TableActionColumnApi; event: MouseEvent }): void
+  (
+    event: 'mouse-event:value',
+    value: { row: TableRow; column: TableColumnApi | TableActionColumnApi; event: MouseEvent },
+  ): void
+}>()
+
+const visibleColumns = computed(() => props.columns.filter((x) => !isSelectionColumn(x)))
+
+function select() {
+  if (!props.selectable) {
+    return
+  }
+
+  emit('select')
+}
+</script>
+
+<template>
+  <div
+    class="s-table-card"
+    :class="{
+      'cursor-pointer': selectable,
+      's-table-card_selected': selected,
+    }"
+    tabindex="0"
+    @click="select"
+    @keypress.space="select"
+  >
+    <div class="p-8px grid grid-cols-[min-content,1fr] items-baseline">
+      <template
+        v-for="(column, columnIndex) in visibleColumns"
+        :key="column.id"
+      >
+        <!-- eslint-disable-next-line vuejs-accessibility/mouse-events-have-key-events vuejs-accessibility/click-events-have-key-events -->
```

Comment:
You turned off a11y lints. Do you handle their reason in JS?

Would be nice to have an explanation as a comment near `eslint-disable` comments.

Reply:
Added comment about events and made table controllable from keyboard

Reply:
Old library have this events which are proxy for some native events but with some component info, so I added them. In terms of accessibility everything is ok because key control for component actions is implemented where needed